### PR TITLE
O3-1416: Fix error upon submission of vitals form 

### PR DESCRIPTION
**Pull request by nanfuka**
[##](url) Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [x] My work includes tests or is validated by existing tests.

## Summary
- Fixes the save for biometrics and vitals
- Adds validations to the vitals form

## Screen Recording
https://user-images.githubusercontent.com/29108523/197728880-dee33f11-8afd-4d73-8a1c-e0a561f91ae8.mp4

## Related Issue
[Cannot Save Vitals and Biometrics](https://issues.openmrs.org/browse/O3-1416)

### CODE REVIEW

**Review by nanfuka — COMMENTED** [review]
@denniskigen, @gracepotma , kindly review

**Comment by gracepotma** [comment]
Thank you Deb! Can you please include a screen recording showing the updated behavior on your local? That makes it much easier for me to review and leave helpful comments :-)

**Comment by nanfuka** [comment]
@gracepotma i have included it. thanks

File: packages/esm-patient-vitals-app/src/vitals/vitals-biometrics-form/vitals-biometrics-form.test.tsx [hunk 1]
```diff
@@ -174,7 +174,7 @@ describe('VitalsBiometricsForm: ', () => {
       message: 'Internal Server Error',
       response: {
         status: 500,
-        statusText: 'Internal Server Error',
+        statusText: 'Some of the values entered are invalid',
```
Comment (hadijahkyampeire) [inline packages/esm-patient-vitals-app/src/vitals/vitals-biometrics-form/vitals-biometrics-form.test.tsx:177]: This should also use the translation
```suggestion
        statusText: t('checkForValidity', 'Some of the values entered are invalid'),
```

**Review by hadijahkyampeire — APPROVED** [review]
Thanks, @nanfuka for this, just a minor suggestion.

File: packages/esm-patient-vitals-app/src/vitals/vitals-biometrics-form/vitals-biometrics-form.test.tsx [hunk 2]
```diff
@@ -174,7 +174,7 @@ describe('VitalsBiometricsForm: ', () => {
       message: 'Internal Server Error',
       response: {
         status: 500,
-        statusText: 'Internal Server Error',
+        statusText: t('checkForValidity', 'Some of the values entered are invalid'),
```
Comment (hadijahkyampeire) [inline packages/esm-patient-vitals-app/src/vitals/vitals-biometrics-form/vitals-biometrics-form.test.tsx:177]:
```suggestion
        statusText: 'Some of the values entered are invalid',
```
Reverting this, we actually don't need it in the test, sorry!

**Pull request merged**